src/main/java/life/qbic/expdesign/io/MHCLigandDesignReader.java

```diff
@@ -138,7 +139,32 @@ public MHCLigandDesignReader() {
     headersToTypeCodePerSampletype.put(SampleType.Q_MHC_LIGAND_EXTRACT, ligandsMetadata);
     // headersToTypeCodePerSampletype.put("Q_MS_RUN", msRunMetadata);
   }
+  private void fillParsedCategoriesToValuesForRow(Map<String, Integer> headerMapping,
```

Comment:
What is a header mapping?

Reply:
the header of the import can be unordered. the map maps col names to positions in the imported header.